proxy/envoy/config.go

```diff
@@ -175,23 +180,24 @@ func buildHTTPListener(mesh *proxyconfig.ProxyMeshConfig, routeConfig *HTTPRoute
 		config.RouteConfig = routeConfig
 	}
 
-	listener := &Listener{
-		Address: fmt.Sprintf("tcp://%s:%d", ip, port),
+	return &Listener{
+		BindToPort: true,
```

Comment:
it's actually specifically unset above. I think it's worth keeping the default explicitly set (missing bindtoport is "true")